scripts/portconfig

```diff
@@ -280,6 +292,10 @@ def main():
                         help = 'port advertised interface types', default=None)
     parser.add_argument('-lt', '--link-training', type = str, required = False,
                         help = 'port link training mode', default=None)
+    parser.add_argument('-P', '--tx-power', type=float, required=False,
```

Comment:
>float [](http://example.com/codeflow?start=49&length=5)

yang model mentioned "fraction-digits 1", but there is not enforcement here. Which one is ground truth? Could you make them consistent?

Reply:
@qiluo-msft, "fraction-digit" in yang-model is correct. do we have any type to match "fraction-digit" in click argument parser?

Reply:
@qiluo-msft added check now